### ISSUE TITLE
No <img> tags have alt attributes

### ISSUE THREAD

**Issue by JakeLabate** [JakeLabate/Strang-HVAC-Demo#13]
Including `alt` attribute values in `<img>` tags improves image search-ability, as well as provides text to show in the even that image for whatever cannot render to the page.

Template:
```html
<img alt="Strang HVAN Van" src="images/strang-van.png">
```
